adapter/src/main/java/org/jvalue/ods/adapterservice/adapter/importer/HttpImporter.java

```diff
@@ -49,6 +53,10 @@ protected void validateParameters(Map<String, Object> inputParameters) {
     return parameters;
   }
 
+  @Override
+  protected List<ImporterParameterDescription> getRequiredParameters() {
```

Comment:
I think we can do it that way!
Another possibility would be to add a boolean field to `ImpoerterParameterDescription` that states if the parameter is required or not. Actually I think that is a good idea, then we could also default other parameters like encoding with a value. But probably better subject of another PR.

Reply:
Okay, good point. I will put it in an upcoming PR.